e2e/testsuite/testsuite.go

```diff
@@ -157,6 +156,18 @@ func (s *E2ETestSuite) SetupChainsRelayerAndChannel(ctx context.Context, channel
 	return r, chainAChannels[len(chainAChannels)-1]
 }
 
+// getPathName generates the path name using the test suites name
+func (s *E2ETestSuite) getPathName() string {
+	pathName := fmt.Sprintf("%s-path", s.T().Name())
+	return strings.ReplaceAll(pathName, "/", "-")
+}
+
+// SetupClients creates clients on chainA and chainB using the provided create client options
+func (s *E2ETestSuite) SetupClients(ctx context.Context, relayer ibc.Relayer, opts ibc.CreateClientOptions) {
+	err := relayer.CreateClients(ctx, s.GetRelayerExecReporter(), s.getPathName(), opts)
+	s.Require().NoError(err)
+}
```

Comment:
I think this is currently not working. My guess is it is no-oping because a client already exists for the given path name. I think we need to generate/add a path for every client/connection/channel (aka endpoint) stack. My vote would be to add the endpoint type and when you call NewEndpoint, it adds the ibctest path name

Reply:
Soo looks like you're right, that new client is not being created. 
The easy solution here is to wire up the `--override` flag in ibctest for the relayer.

The client is not begin created b/c in the relayer config, the client already exists so it does not create a new client.
This is what the `--override` flag is used for; to create a new client even though the config is showing a client-id already exists. But, it is currently not working. Issue here: https://github.com/cosmos/relayer/issues/952

In the meantime, one dirty solution that I *think* will work is to create another relayer instances with a different home folder path. Then create the new "substituteClient" with the bad trusting period that way.

Reply:
Generating a new path (with a new path name) worked